From 92b5a40d2d84e1989cdd091a10660e14e66c4daf Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <mart@leio.tech>
Date: Sun, 16 Jan 2022 01:03:08 +0200
Subject: [PATCH 3/3] build: Use API version in gettext domain

Avoids file conficts with other API versions being installed on the system.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 00cfbeec..db44e1dd 100644
--- a/meson.build
+++ b/meson.build
@@ -78,7 +78,7 @@ libgucharmap_gtk4_soversion = '0'
 
 # i18n
 
-gucharmap_gettext_domain = gucharmap_name
+gucharmap_gettext_domain = libgucharmap_gtk3_api_name
 po_dir = meson.current_source_dir() / 'po'
 
 # Directories
-- 
2.32.0

